packages/yumemi_lints/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Examples of version updates are as follows:
 
 - Changed the warning level of `invalid_use_of_visible_for_testing_member` to error.
 - Changed the lint rule `use_setters_to_change_properties` to be deprecated.
+- Remove the patch version from the directory name of Lint Rules.
```

Comment:
![nits-badge](https://img.shields.io/badge/review-nits-orange.svg)
```suggestion
- Removed the patch version from the directory name of Lint Rules.
```

Reply:
I should have used the past tense, my apologies.
I fixed. 6310aadeb4fb3cafc37c43d5f7e3ee4f73cd991c